configurations/dadaab_pipeline_config.json

```diff
@@ -0,0 +1,81 @@
+{
+  "RawDataSources": [
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/wusc-keep-II-dadaab.textit-token.txt",
```

Comment:
We've used a `-` instead of a `.` in the past...

Reply:
updated thanks.